the the

```diff
@@ -692,7 +692,8 @@ You likely want to use the `DocsContainer` component exported by `@storybook/blo
 
 **Overriding theme**:
 
-To override the theme, you can continue to use the `docs.theme` parameter.
+To override 
+me, you can continue to use the `docs.theme` parameter.
```

Comment:
@mohdashraf010897, I'd appreciate it if you wouldn't mind reverting this to how it was initially. The change you're making here is introducing additional typos.